src/helpers.js

```diff
@@ -32,40 +32,42 @@ const CONSOLE_API_KEYS = {
  * @returns {any} Returns a config object or null
  */
 async function getCommerceAdminConfig() {
-	const configFile = Config.get('aio-cli-plugin-commerce-admin');
+	const configFile = Config.get('mesh-api-manager.configPath');
 
 	if (!configFile) {
 		return {
-			baseUrl: 'https://graph.adobe.io/api-admin',
+			baseUrl: 'https://developers.adobe.io/console',
```

Comment:
Shouldn't this be https://developer.adobe.io/console

Reply:
The URL I was given had a `s` next to developer in the URL